geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -25,160 +25,320 @@ optional password authentication.
+- MSETNX
+- PERSIST
+- PEXPIRE
+- PEXPIREAT
+- PING
+- PSETEX
+- PSUBSCRIBE
+- PTTL
+- PUBLISH
+- PUBSUB
+- PUNSUBSCRIBE
+- RENAME
+- RENAMENX
+- RESTORE
+- SADD
+- SCARD
+- SDIFF
+- SDIFFSTORE
+- SET
+- SETEX
+- SETNX
+- SETRANGE
+- SINTER
+- SINTERSTORE
+- SISMEMBER
+- SLOWLOG **[5]**
+- SMEMBERS
+- SMOVE
+- SPOP
+- SRANDMEMBER
+- SREM
+- SSCAN
+- STRLEN
+- SUBSCRIBE
+- SUNION
+- SUNIONSTORE
+- TTL
+- TYPE
+- UNSUBSCRIBE
+- QUIT
+- ZADD
+- ZCARD
+- ZCOUNT
+- ZINCRBY
+- ZINTERSTORE
+- ZLEXCOUNT
+- ZPOPMAX
+- ZPOPMIN
+- ZRANGE
+- ZRANGEBYLEX
+- ZRANGEBYSCORE
+- ZRANK
+- ZREM
+- ZREMRANGEBYLEX
+- ZREMRANGEBYRANK
+- ZREMRANGEBYSCORE
+- ZREVRANGE
+- ZREVRANGEBYLEX
+- ZREVRANGEBYSCORE
+- ZREVRANK
+- ZSCAN
+- ZSCORE
+- ZUNIONSTORE
 
-## <a id="supported-commands" class="no-quick-link"></a>Supported Redis Commands
 
-<%=vars.product_name%> for Redis supports the following Redis commands.
-<br/>
-
- - APPEND <br/>
- - AUTH <br/>
- - DECR <br/>
- - DECRBY <br/>
- - DEL <br/>
- - EXISTS <br/>
- - EXPIRE <br/>
- - EXPIREAT <br/>
- - GET <br/>
- - GETRANGE <br/>
- - HDEL <br/>
- - HEXISTS <br/>
- - HGET <br/>
- - HGETALL <br/>
- - HINCRBY <br/>
- - HINCRBYFLOAT <br/>
- - HLEN <br/>
- - HMGET <br/>
- - HMSET <br/>
- - HSCAN **[1]**  <br/>
- - HSET <br/>
- - HSETNX <br/>
- - HSTRLEN <br/>
- - HVALS <br/>
- - HKEYS <br/>
- - INCR <br/>
- - INCRBY <br/>
- - INCRBYFLOAT <br/>
- - INFO **[2]**  <br/>
- - KEYS <br/>
- - MGET <br/>
- - PERSIST <br/>
- - PEXPIRE <br/>
- - PEXPIREAT <br/>
- - PING <br/>
- - PSUBSCRIBE <br/>
- - PTTL <br/>
- - PUBLISH <br/>
- - PUNSUBSCRIBE <br/>
- - QUIT <br/>
- - RENAME <br/>
- - SADD <br/>
- - SCARD <br/>
- - SDIFF <br/>
- - SDIFFSTORE <br/>
- - SINTER <br/>
- - SISMEMBER <br/>
- - SET <br/>
- - SETNX <br/>
- - SLOWLOG **[3]**  <br/>
- - SMEMBERS <br/>
- - SMOVE <br/>
- - SREM <br/>
- - STRLEN <br/>
- - SUBSCRIBE <br/>
- - SUNION <br/>
- - TTL <br/>
- - TYPE <br/>
- - UNSUBSCRIBE <br/>
-
-<br/>
 Commands not listed above are **not implemented**.
 
-<br/>
 **NOTES:**
 
 These commands are supported for Redis 5.
+**[1]]** CLUSTER is implemented for the subcommands INFO, NODES, SLOTS, and KEYSLOT.
 
-**[1]** Redis accepts 64-bit signed integers for the HSCAN cursor and COUNT parameters.
- <%=vars.product_name%> for Redis is limited to 32-bit integer values for these parameters.
+**[2]** COMMAND is implemented only with no subcommands.
 
-**[2]** INFO is implemented for the sections and fields listed below:
+**[3]** Redis accepts 64-bit signed integers for the HSCAN cursor and COUNT parameters.
+<%=vars.product_name%> for Redis is limited to 32-bit integer values for these parameters.
 
- - clients
+**[4]** INFO is implemented for the sections and fields listed below:
 
-    - connected_clients
+- clients
 
-    - blocked_clients (always returns 0)
+   - connected_clients
 
- - cluster
+   - blocked_clients (always returns 0)
 
-    - cluster_enables (always returns 0)
+- cluster
 
- - keyspace
+   - cluster_enables (always returns 0)
```

Comment:
This no longer always returns 0, but instead always returns 1, because Geode for Redis always runs in cluster mode.